### DISCUSSION TITLE
Enzyme expression database : problems in V11

### DISCUSSION THREAD

**Discussion by adinizbe** [Open-Systems-Pharmacology/Forum#1282] [Questions & Problems]
Hi
I just downloaded the new version of PKSim and when I tried to create an individual, the addition of any enzyme did not show the relative expression in each tissue. How can I fix it? Please?

**Comment by adinizbe** [comment]
![image](https://user-images.githubusercontent.com/52289547/184969917-4d802605-32ff-45bc-a0a3-6e5929b2222d.png)

**Comment by Yuri05** [comment]
You must either set relative expressions manually or load them from the Gene Expression database:
https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-expression-profile#editing-an-expression-profile-in-pk-sim-r

**Comment by adinizbe** [comment]
Hi, Thank you for your answer.
But I already downloaded, as usual, but did not work, for my suprise.

**Comment by StephanSchaller** [comment]
Hi Andrea,

I would recommend to start with some training on the use of the expression database in PK-Sim. For academia, you can find free learning material on our online platform: register (free: https://learn-esqlabs.talentlms.com/) and select the course "The OSPS Expression Database"
